docs/admin/install/steps/install-configure.rst

```diff
@@ -3,10 +3,9 @@ Configuring Weblate
 
 .. note::
 
-   Following steps assume virtualenv used by Weblate is active (what can be
-   done by ``. ~/weblate-env/bin/activate``). In case this is not true, you will
-   have to specify full path to :command:`weblate` command as
-   ``~/weblate-env/bin/weblate``.
+   The following assumes the virtualenv used by Weblate is running
```

Comment:
```suggestion
   The following assumes the virtualenv used by Weblate is started
```
Slightly less awkward sentence. Might be more ways to start it though.
Wanted to avoid "active" as possible meaning many things.

Reply:
It is neither not running nor started, it is activated. Please stop inventing new terms here.

See https://virtualenv.pypa.io/en/latest/user_guide.html#activators

Reply:
The problem arises when the word has other meanings, and is made worse when the context is blurry and the sentence structure alludes to its other uses. Likely translations will then reflect these, in languages where it is not shared.
The core problem doesn't need to be fixed, but now it is clear for contextual documentation, and then it has a chance of working or for someone to find the time to arrive at a better solution.